EcalTiming/plugins/EcalTimingCalibProducer.cc

```diff
@@ -505,9 +519,10 @@ EcalTimingCalibProducer::Status EcalTimingCalibProducer::duringLoop(const edm::E
 #endif
 	// If we got less than the minimum recHits, continue
 	if(eventTimeMap_.size() < _recHitMin) return kContinue;
-	if(timeEB.num() < 4) return kContinue;
-	if(timeEEM.num() < 4 && timeEEP.num() < 4) return kContinue;
+	if(timeEB.num() + timeEEM.num() + timeEEP.num() < _minEntries) return kContinue;
```

Comment:
This is originally from Peter. I think one is the minimum number of rechits in the collection, the second is the minimum number of entries in the final map to make the averages (i.e. after the selections).